osfoffline/utils/__init__.py

```diff
@@ -5,18 +5,18 @@
 import logging.config
 import logging.handlers
 import os
-import sys
```

Comment:
Please add this back. This removal breaks the custom `excepthook` on line 22.

(sys.excepthook is used as a quick-and-dirty way to ensure that all uncaught exceptions get logged)